page/partials/commit_funds.ejs

```diff
@@ -13,12 +13,12 @@
 
             <div class="col-md-offset-2 col-md-5">
                 <div class="number">
-                    <%- sections[0].point -%>
+                    <%- sections[1].point -%>
                 </div>
                 <div class="content">
-                    <h3><%- sections[0].title -%></h3>
-                    <p><%- sections[0].paragraph1 %></p>
-                    <p><%- sections[0].paragraph2 %></p>
+                    <h3><%- sections[1].title -%></h3>
+                    <p><%- sections[1].paragraph1 %></p>
+                    <p><%- sections[1].paragraph2 %></p>
```

Comment:
I don't think that's a good idea take notice that styles might change any minute to something completely different, they are identical now but its not guaranteed. Doing something like this might create unneeded over head